src/style/components/mls-device-details.less

```diff
@@ -0,0 +1,16 @@
+.mls-device-details {
+  width: 100%;
+
+  .preferences-devices & {
+    padding-bottom: 32px;
+  }
+
+  .preferences-device-details &,
+  .participant-devices__header & {
+    border-bottom: 1px solid var(--gray-40);
+  }
+
+  .participant-devices__header & {
+    padding-bottom: 24px;
+  }
+}
```

Comment:
When adding new styles, please add them to a new `Component.styles.ts` file colocated with the component.